index.html

```diff
@@ -358,53 +357,40 @@ <h2 id="3Definitions">Definitions</h2>
 <dt><dfn id="3alphaCompaction">alpha compaction</dfn></dt>
 
 <dd>an implicit representation of transparent
-<span class="Definition"><a>pixels</a></span>. If every
-pixel with a specific colour or <span
-class="Definition"><a>greyscale</a></span> value is fully
+<a>pixels</a>. If every
```

Comment:
Does respec deal with singular and plural? (I know bikeshed does, or tries to, not always successfully)? Here `pixel` and `pixels` should link to the same definition.

Reply:
The docs say it does.

Reply:
Cool, great!